src/submission/submission-error-messages.ts

```diff
@@ -59,7 +59,7 @@ const ERROR_MESSAGES: { [key: string]: (errorData: any) => string } = {
 const BATCH_ERROR_MESSAGES: Record<SubmissionBatchErrorTypes, (errorData: any) => string> = {
   [SubmissionBatchErrorTypes.TSV_PARSING_FAILED]: () => `Failed to parse the tsv file`,
   [SubmissionBatchErrorTypes.INVALID_FILE_NAME]: () =>
-    `Please retain the template file name and only append characters to the end. For example, sample_registration<_optional_extension>.tsv`,
+    `Improperly named files cannot be uploaded or validated. All files must start with the corresponding entity name and have the .tsv extension (e.g. donor<_optional_extension>.tsv).`,
```

Comment:
One thing to note, this message is used for both registration and submission....

Reply:
Don't know if we wanted to change it for both. If not, maybe you could pass the clinical type into the batchError builder and use the clinical type to figure out what the message should say

Reply:
@rosibaj @kcullion  

Should there be a separate message for submission vs clinical? Or is it common to all?

Reply:
@wajiha-oicr I would say yes, except in sample registration it would be nice to for the example to say (e.g. sample_registration<_optional_extension>.tsv).

Reply:
@jaserud  https://github.com/icgc-argo/argo-clinical/pull/377/commits/121a53fe6070d625d93e0dea390b222e01a2b129